session_test.go

```diff
@@ -1179,7 +1179,7 @@ var _ = Describe("Session", func() {
 			sess.ackAlarmChanged(time.Now().Add(10 * time.Millisecond))
 			time.Sleep(10 * time.Millisecond)
 			Eventually(func() int { return len(mconn.written) }).ShouldNot(BeZero())
-			Expect(mconn.written[0]).To(ContainSubstring(string([]byte{0x37, 0x13})))
+			Eventually(func() []byte { return mconn.written[0] }).Should(ContainSubstring("foobar"))
```

Comment:
I don't really understand what we're testing here, and why it is valid to not test for the ACK anymore. Can we find a more meaningful name for the test?